readme.md

```diff
@@ -1,12 +1,12 @@
 # ava-codemods [![Build Status](https://travis-ci.org/avajs/ava-codemods.svg?branch=master)](https://travis-ci.org/avajs/ava-codemods)
 
-> Codemods for [AVA](https://ava.li) that simplifies upgrading to newer versions
+> Codemods for [AVA](https://ava.li) that simplifies upgrading to newer versions and migrating to AVA
 
 <img src="screenshot.gif" width="440" align="right">
 
 Codemods are small programs that help you automate changes to your codebase. Think of them as search and replace on steroids.
 
-This module contains a set of codemods that enable you to upgrade your code between various AVA releases. It is maintained by the AVA team, and will be updated anytime we introduce breaking API changes. We plan to eventually introduce codemods that allow you to switch from other popular test runners like `mocha` and `tap`.
+This module contains a set of codemods that enable you to upgrade your code between various AVA releases and migrate from existing  test runners to AVA. It is maintained by the AVA team, and will be updated anytime we introduce breaking API changes. 
```

Comment:
Typo nitpick: double space in `from existing  test runners to AVA`.
(But I/someone else can fix them later, let this not be a blocker)